src/app/engineering/blog/page.tsx

```diff
@@ -0,0 +1,8 @@
+import Blog from "@/components/engineering/Blog";
+import React from "react";
```

Comment:
![image](https://github.com/user-attachments/assets/1322c588-dd1d-4d53-9db0-510bcd1d8337)
should be similar to this